vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -348,6 +348,35 @@ public String getCodeGrantAuthorizationURI() {
     }
 
     /**
+     * Authorizes users of the app who are not signed in. This call requires a client id and client secret
+     * to be set on the initial Configuration.
+     * <p/>
+     * Leaves User as null in {@link VimeoAccount} model and populates the rest.
+     * <p/>
+     * WARNING: This contains a synchronous network call. Use {@link #authorizeWithClientCredentialsGrant(AuthCallback)}
+     * for asynchronous use.
+     */
+    public VimeoAccount authorizeWithClientCredentialsGrantSync() {
+
+        Call<VimeoAccount> call = mVimeoService.authorizeWithClientCredentialsGrant(getBasicAuthHeader(),
+                                                                                    Vimeo.CLIENT_CREDENTIALS_GRANT_TYPE,
+                                                                                    mConfiguration.scope);
+
+        VimeoAccount vimeoAccount = null;
+        try {
+            retrofit2.Response<VimeoAccount> response = call.execute();
+             if (response.isSuccessful()) {
+                vimeoAccount = response.body();
+                saveAccount(vimeoAccount, null);
```

Comment:
We might want to pull this block out into a helper just so it's consistent with the async version:

```java
            if (vimeoAccount.getUser() != null && (mEmail == null || mEmail.isEmpty())) {
                // We must always have a `name` field, which is used by the Android Account Manager for
                // display in the device Settings -> Accounts [KZ] 12/17/15
                String name = (vimeoAccount.getUser().name !=
                               null) ? vimeoAccount.getUser().name : vimeoAccount.getUser().uri;
                mClient.saveAccount(vimeoAccount, name);
            } else {
                mClient.saveAccount(vimeoAccount, mEmail);
            }
```